theme-shop/boss.py

```diff
@@ -0,0 +1,124 @@
+from pyctr.crypto.engine import CryptoEngine, Keyslot
+from io import BytesIO
+from typing import IO
+import os
+
+def readle(i: IO, s) -> int:
+	return int.from_bytes(i.read(s), 'little')
+
+def readbe(i: IO, s) -> int:
+	return int.from_bytes(i.read(s), 'big')
+
+class BOSSHeader:
+	magic: str
+	magic_num: int
+	filesize: int
+	unk_int: int # BE bytes (well not BE but read as BE), does not seem to be a date
```

Comment:
```suggestion
	unk_int: int # * BE bytes (well not BE but read as BE), does not seem to be a date
```

We use the comment syntax from [Better Comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments) here

Only going to leave the one comment despite there being multiple instances of this

Reply:
Should be good now.